mkdocs.yml

```diff
@@ -0,0 +1,37 @@
+site_name: 'MLBox'
+site_description: 'Documentation site for MLBox'
+site_author: 'Adhita Selvaraj <adhita.selvaraj@gmail.com>'
```

Comment:
I suggest we use an email `mlperf@googlegroups.com`
already being used in other projects >https://github.com/mlperf/logging/blob/master/setup.py#L27

Reply:
Changed